### PR TITLE
add gql 

### PR DESCRIPTION
**Pull request by ifielder**

### CODE REVIEW

File: pages/api/db/index.ts [hunk 1]
```diff
@@ -0,0 +1,118 @@
+import * as dynamoose from 'dynamoose'
+import { Document } from 'dynamoose/dist/Document'
+
+type AnimeTableAttributes = { id: string; entity: string }
+
+dynamoose.aws.sdk.config.update({
+  accessKeyId: process.env.AWSACCESSKEYID,
```
Comment (m1yon) [inline pages/api/db/index.ts:7]:
```suggestion
  accessKeyId: process.env.AWS_ACCESS_KEY_ID,
```

File: pages/api/db/index.ts [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+  secretAccessKey: process.env.AWSSECRETACCESSEY,
```
Comment (m1yon) [inline pages/api/db/index.ts:8]:
```suggestion
  secretAccessKey: process.env.AWS_SECRET_ACCESS_KEY,
```

File: pages/api/db/index.ts [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+  region: process.env.AWSREGION,
```
Comment (m1yon) [inline pages/api/db/index.ts:9]:
```suggestion
  region: process.env.AWS_REGION,
```

File: typings/graphql/types.d.ts [hunk 4]
```diff
@@ -0,0 +1,62 @@
+type Maybe<T> = T | null;
```
Comment (m1yon) [inline typings/graphql/types.d.ts:1]: do you want this committed if it's generated?

**Review by m1yon — APPROVED** [review]

File: typings/graphql/types.d.ts (hunk 4 above)
Comment (ifielder) [reply typings/graphql/types.d.ts:1]: no

**Pull request merged**